tools/mksurfdata_esmf/src/mkagfirepkmonthMod.F90

```diff
@@ -116,6 +120,10 @@ subroutine mkagfirepkmon(file_mesh_i, file_data_i, mesh_o, pioid_o, rc)
     call mkpio_get_rawdata(pioid_i, 'abm', mesh_i, idata_i, rc=rc)
     if (chkerr(rc,__LINE__,u_FILE_u)) return
     call ESMF_VMLogMemInfo("After mkpio_getrawdata in "//trim(subname))
+    ! Update idata_i to unsetmon where mask_i == 0, i.e. over ocean
+    do ni = 1, ns_i
+       if (mask_i(ni) == 0) idata_i(ni) = unsetmon
```

Comment:
@samsrabin @lifang0209 
I brought back unsetmon as 14 to distinguish ocean points and thus remove them from the determination of the dominant abm. I think it makes more sense than allowing the lack of peak month over oceans to skew the result.

Reply:
Fang, you can see the new fsurdat here:
/glade/campaign/cesm/cesmdata/cseg/inputdata/lnd/clm2/surfdata_esmf/ctsm5.4.0/surfdata_1.9x2.5_hist_2000_16pfts_c250617.nc